client/options_internal_test.go

```
@@ -81,13 +81,17 @@
 	}
 	etag := "123"
 	codec := ocf.VNDOCFCBORCodec{}
+	linkNotFoundCallback := func(links schema.ResourceLinks, href string) (schema.ResourceLink, error) {
```

golangci-lint

[warning] unused-parameter: parameter 'links' seems to be unused, consider removing or renaming it as _

(revive)